pkg/internal/checkup/pod/container.go

```diff
@@ -107,6 +107,15 @@ func WithContainerHugepagesVolumeMount() ContainerOption {
 	}
 }
 
+func WithContainerLibModulesVolumeMount() ContainerOption {
+	return func(container *corev1.Container) {
+		container.VolumeMounts = append(container.VolumeMounts, corev1.VolumeMount{
+			Name:      "modules",
+			MountPath: "/lib/modules",
```

Comment:
Please consider adding the `ReadOnly: Pointer(true)` option.

Reply:
DONE